### ISSUE TITLE
Non-2D Softmax/LogSoftmax are wrong

### ISSUE THREAD

**Issue by shinh** [chainer/onnx-chainer#67]
Chainer's `F.softmax` is different from ONNX's

See: https://github.com/shinh/onnx-chainer/commit/bf170a95638fb5fd59adfc9c90d765581dd89195

**Comment by disktnk** [comment]
fixed by #174
